src/bundle/DependencyInjection/SchebTwoFactorExtension.php

```diff
@@ -36,26 +36,30 @@ public function load(array $configs, ContainerBuilder $container): void
         }
 
         // Load two-factor modules
-        if (isset($config['email']['enabled']) && true === $config['email']['enabled']) {
+        if (isset($config['email']['enabled']) && true === $this->resolveFeatureFlag($container, $config['email']['enabled'])) {
```

Comment:
The `true === ` can be removed. `resolveFeatureFlag` is only returning booleans, so checking for `true` is redundant here.